subxt/src/backend/unstable/rpc_methods.rs

```diff
@@ -745,11 +769,16 @@ pub enum TransactionStatus<Hash> {
 
 /// Details of a block that a transaction is seen in.
 #[derive(Debug, Clone, PartialEq, Eq, Deserialize)]
+#[cfg_attr(test, derive(Serialize))]
 pub struct TransactionBlockDetails<Hash> {
     /// The block hash.
     pub hash: Hash,
     /// The index of the transaction in the block.
-    #[serde(with = "unsigned_number_as_string")]
+    #[serde(deserialize_with = "unsigned_number_as_string::deserialize")]
+    #[cfg_attr(
+        test,
+        serde(serialize_with = "unsigned_number_as_string::for_test::serialize")
```

Comment:
aight, I see makes sense to have these in this module then

Missed the serialize_with